stories/2022-10-06-Boris-Bikes/requirements.txt

```diff
@@ -0,0 +1,134 @@
+anyio==3.6.1; python_full_version >= "3.6.2" and python_version >= "3.7"
```

Comment:
In order to get the notebook to work with pip I had to make the following changes:
```diff
diff --git a/stories/2022-10-06-Boris-Bikes/requirements.txt b/stories/2022-10-06-Boris-Bikes/requirements.txt
index 5fb5347..492a415 100644
--- a/stories/2022-10-06-Boris-Bikes/requirements.txt
+++ b/stories/2022-10-06-Boris-Bikes/requirements.txt
@@ -30 +30 @@ fastjsonschema==2.16.1; python_version >= "3.7"
-fiona==1.8.21; python_version >= "3.8"
+fiona>=1.8.21; python_version >= "3.8"
@@ -61 +61 @@ matplotlib-inline==0.1.3; python_version >= "3.8"
-matplotlib==3.5.2; python_version >= "3.7"
+matplotlib>=3.5.2; python_version >= "3.7"
@@ -95 +95 @@ pyparsing==3.0.9; python_full_version >= "3.6.8" and python_version >= "3.7"
-pyproj==3.3.1; python_version >= "3.8"
+pyproj>=3.3.1; python_version >= "3.8"
@@ -105 +105 @@ pyyaml==6.0; python_version >= "3.6"
-pyzmq==23.2.0; python_version >= "3.7"
+pyzmq>=23.2.0; python_version >= "3.7"
```